### PR TITLE
Add GETs for gateway and virtual machine resources

### PR DESCRIPTION
**Pull request by ChristianPomales**
Add GETs for gateway and virtual machine resources.

### CODE REVIEW

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json [hunk 1]
```diff
@@ -2582,6 +2643,51 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/workloadNetworks/default/virtualMachines/{virtualMachineId}": {
+      "get": {
+        "operationId": "WorkloadNetworks_GetVirtualMachines",
```
Comment (ctaggart) [inline specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:2649]: Probably remove the `s.

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json [hunk 2]
```diff
@@ -2034,6 +2050,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/workloadNetworks/default/gateways/{gatewayId}": {
+      "get": {
+        "operationId": "WorkloadNetworks_GetGateways",
```
Comment (ctaggart) [inline specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:2055]: Probably remove the `s`.

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json [hunk 3]
```diff
@@ -2034,6 +2050,51 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkloadNetworks_ListGateways": {
+            "$ref": "./examples/WorkloadNetworks_GetGateways.json"
```
Comment (ctaggart) [inline specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:2093]: Here too.

**Review by ctaggart — CHANGES_REQUESTED** [review]
The operation id category is plural, but not the operation. Please drop the trailing `s`s.

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json (hunk 2 above)
Comment (ChristianPomales) [reply specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json]: Fixed

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json (hunk 3 above)
Comment (ChristianPomales) [reply specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json:2093]: Fixed

File: specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json (hunk 1 above)
Comment (ChristianPomales) [reply specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json]: Fixed

**Pull request merged**